channels/management/commands/reclassify_spam.py

```diff
@@ -65,12 +66,16 @@ def handle(self, *args, **options):
             self.stdout.write("Either --spam or --ham flag is required")
             return
 
-        task = tasks.update_spam.delay(
-            spam=is_spam,
-            comment_ids=options["comment_ids"],
-            post_ids=options["post_ids"],
-            retire_users=options["retire_users"],
-            skip_akismet=options["skip_akismet"],
+        task = tasks.update_spam.apply_async(
```

Comment:
I think it'd be better to use [automatic routing](https://docs.celeryproject.org/en/stable/userguide/routing.html#automatic-routing) since you don't have to worry about covering all of the task call sites now or in the future. It's probably not much of a gain right now for this work, but later on if we start splitting other tasks into their own queues we'll want one centralized place to designate the routing.